csrc/scheduler/mma_utils.cpp

```diff
@@ -703,11 +704,51 @@ std::unordered_set<IterDomain*> getMmaDomainSet(
   return {mma_domains.begin(), mma_domains.end()};
 }
 
+bool isConsumerAllocationInnerIDProducerAllocationInnerID(
+    TensorView* consumer) {
+  // We are only looking at the allocation domains accross
+  // the producer and consumer of a LoadStoreOp
+  auto LdStOp = consumer->definition();
+  NVF_CHECK(dynamic_cast<LoadStoreOp*>(LdStOp));
```

Comment:
Does the definition need to be a LoadStoreOp for this utility?

Reply:
That's a really good point - I don't think this I have to limit it to a LoadStoreOp. I'll make that change.